client/dist/styles/reviews.css

```diff
@@ -122,4 +123,34 @@
   left: 0;
   bottom: 0;
   right: 0;
-}
\ No newline at end of file
+}
+
+input[type='radio'] {
```

Comment:
this is an interesting CSS syntax. Would you mind sharing what the brackets target in this case? I'm curious.